### ISSUE TITLE
[Bug]: FTB Quests: Quest Icons with a size of 2.0 Overlap Certain UI Elements

### ISSUE THREAD

**Issue by alfredggttv** [FTBTeam/FTB-Mods-Issues#1320]
### Mod

FTB Quests

### Mod version

ftb-quests-neoforge-2100.1.4

### Forge / Fabric version

NeoForge 21.1.8

### Modpack & version

FTB Flagship

### What issue are you having?

When changing the icon size of a quest to 2.0, the icon begins to overlap parts of the menu as seen below.
![image](https://github.com/user-attachments/assets/4b9da300-90b2-45ec-a6ef-eace1ac77775)

This also makes it harder for people to see if they have complete the quests because the icons cover up the Checkmark for completion:
![image](https://github.com/user-attachments/assets/6b43e21d-6167-4025-bde3-6d740a597ada)

### Steps to reproduce

1. In a quest chapter, set an icon's size to 2.0. Anything below 2.0 does not cause the bug.
2. With the chapter list menu opened or pinned, drag the quest page around until the Icon clips/overlaps into the menu.
(Optionally)
3. Complete a quest that has the icon size set to 2. The checkmark will be covered up by the icon.

### Anything else to note?

I believe the checkmarks being covered up has also happened in previous versions, such as 1.20.4.

**Comment by desht** [comment]
Yeah it's an issue with some vanilla changes that got made somewhere in the 1.20.x code... any block items drawn with scaling also have Z scaling, and in GUI context that means they stick out of the screen toward the user.

Can generally be worked around by adjusting the Z-level when other GUI elements are drawn. I'll go back in and review the Z-levels for the side panel and the tick mark and check if anything else needs adjusting.

**Comment by desht** [comment]
This one's actually causing me some trouble, due to tooltips. While it's easy enough to adjust the Z-level of the chapter panel and the icons, adjusting tooltip levels (which are drawn by vanilla) isn't so easy. A straightforward translation on the pose stack z-level doesn't do the trick. So this is going to need some more research, unfortunately. Will see what I can do, though.

**Comment by TheBedrockMaster** [comment]
Was there any update on this?

**Comment by desht** [comment]
Yeah latest version of FTBQ (2101.1.1) should resolve this, hopefully.  Let me know.

**Comment by TheBedrockMaster** [comment]
It's working with ATM10 1.17! 🎉

**Comment by TheBedrockMaster** [comment]
I was having a similar issue again recently.
